src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -355,6 +355,16 @@ public static FormulaValue Time(IRContext irContext, NumberValue[] args)
             return new TimeValue(irContext, result);
         }
 
+        public static FormulaValue DateTimeFunction(IRContext irContext, NumberValue[] args)
+        {
+            var date = Date(IRContext.NotInSource(Core.Public.Types.FormulaType.Date), new NumberValue[] { args[0], args[1], args[2] });
```

Comment:
>args[0] [](http://example.com/codeflow?start=107&length=7)

nit: 
for readability, consider introducing locals:
var year = args[0];
var month = args[1];

Reply:
Done